README.md

```diff
@@ -219,47 +219,42 @@ let () = printGithubRepos ()
 ## Node-specific
 
 #### Read lines from a text file
-
+Uses [bs-node](https://github.com/BuckleTypes/bs-node)
 ```ml
 let () =
-    Node_fs.readFileSync "README.md" `utf8
+    Node.Fs.readFileAsUtf8Sync "README.md"
     |> Js.String.split "\n"
     |> Array.iter Js.log
```

Comment:
The indentation of the new examples aren't consistent with the other examples. You seem to be using 4 spaces instead of 2.